### PR TITLE
re-formatted advisory blockquote, and added advisories to Toolbox docs

### PR DESCRIPTION
**Pull request by londoncalling**
### Proposed changes

Updated format for blockquotes on advisories (gradient grey background w/yellow sidebar).

* The goal is to make the advisories more prominent to flag legacy or specific topics, different from our standard tips and notes.

* Tested on Safari, Chrome, where it works. On Firefox, it shows as non-gradient, default background. Will test on Explorer, mobile docs, other? We can make it work on other browsers if needed, or we can let it use default background.

Added Toolbox advisories.

### Please take a look

@sanscontext @johndmulhausen @mstanleyjones @joaofnfernandes @nathanleclaire 

Signed-off-by: Victoria Bialas <victoria.bialas@docker.com>

### CODE REVIEW

**Comment by johndmulhausen** [comment]
Deploy preview failed.

Built with commit 62edf2ea80a7d44137b73bb871f95260f3fab210

https://app.netlify.com/sites/docker-docs/deploys/5849ea0b6686744da0c91439

File: _data/advisories.yaml [hunk 1]
```diff
@@ -12,6 +12,7 @@ texts:
   rc: "The Swarm mode feature included in Docker Engine 1.12 is a release candidate feature and might be subject to non backward-compatible changes. Some functionality may change before the feature becomes generally available."
   docker4mac-beta: "Docker for Mac is currently in public beta. Some functionality may change before the product becomes generally available."
   docker4win-beta: "Docker for Windows is currently in public beta. Some functionality may change before the product becomes generally available."
+  toolbox: "**Legacy desktop solution.** Docker Toolbox is for older Mac and Windows systems that do not meet the requirements of the new [Docker for Mac](/docker-for-mac/) and [Docker for Windows](/docker-for-windows/) apps. We recommend updating to the newer apps, if possible."
```
Comment (sanscontext) [inline _data/advisories.yaml]: I would maybe `s/apps/desktop applications/g`
But that gets wordy, so that's optional.

**Comment by sanscontext** [comment]
Whoa, but clearly Netlify doesn't agree. o_O

**Comment by londoncalling** [comment]
OK, LGTM for now. We can re-visit this if there are more comments. Thanks @sanscontext .

**Pull request merged**